### PR TITLE
Different Styles in the background slideshow!

### PR DESCRIPTION
**Pull request by vecchiotom**

### CODE REVIEW

File: __resource.lua [hunk 1]
```diff
@@ -22,7 +22,9 @@ files {
     'js/progressbar-renderer.js',
     'js/music-handler.js',
     'js/music-controls.js',
-    'js/background-handler.js',
+    'js/jquery.js',
```
Comment (Syntasu) [inline __resource.lua]: No jquery please, nor jqueryui and cycle.js.

File: css/index.css [hunk 2]
```diff
@@ -13,7 +13,7 @@ body
     margin: 0;
 }
 
-#background
+#slider
```
Comment (Syntasu) [inline css/index.css]: Please keep the original id and adjust it to your needs

File: js/background-handler.js [hunk 3]
```diff
@@ -1,23 +0,0 @@
...
-    index = (index + 1) % config.background.length;
-
-    lib.fadeInOut(600, "background", 0, 0.2);
-
-    setTimeout(function()
-    {
-        background.setAttribute("src", config.background[index]);
-    }, 600);
-}
-
```
Comment (Syntasu) [inline js/background-handler.js]: Why is this removed?

File: index.html [hunk 4]
```diff
@@ -7,9 +7,10 @@
     </head>
 
     <body>
-        <img id="background"/>
+            <img id="cursor" src="img/cursor.png"/>
```
Comment (Syntasu) [inline index.html]: Inconsistent indentation.

File: js/config.js [hunk 5]
```diff
@@ -124,5 +124,19 @@ var config =
...
+    /*
+        effect the background slideshow should have.
+        you can use :
+        - shuffle
+        - fade
+        - zoom
+        - turnDown
+        - curtainX
+     */
+    fx: "shuffle",
```
Comment (Syntasu) [inline js/config.js]: Rename to backgroundEffect please

File: js/config.js [hunk 6]
```diff
@@ -124,5 +124,19 @@ var config =
...
+
+
```
Comment (Syntasu) [inline js/config.js]: Redundant white spaces

File: js/music-controls.js [hunk 7]
```diff
@@ -90,12 +90,21 @@ window.onload = function()
   document.body.addEventListener("mousemove", function(event)
   {
         var cursor = document.getElementById("cursor");
-
+console.log('triggered')
```
Comment (Syntasu) [inline js/music-controls.js]: Remove this log please.

File: js/music-controls.js [hunk 8]
```diff
@@ -90,12 +90,21 @@ window.onload = function()
...
         //TODO: More consistent way of aligning the cursor without awkward offsets?
         var x = event.pageX - cursor.width + 7;
         var y = event.pageY - 7;
 
         cursor.style.left = x;
         cursor.style.top = y;
   });
+  for (let i = 0; i < config.background.length; i++) {
+    document.getElementById('slider').innerHTML+='<img src="'+config.background[i]+'"width="100%" height="100%"/>'
+
```
Comment (Syntasu) [inline js/music-controls.js]: Why is this on the mouse move event?

File: js/synn.js [hunk 9]
```diff
@@ -5,21 +5,22 @@ var lib =
...
-    fadeInOut: function(duration, elementId, min, max)
+    fadeInOut: function(duration, elementId)
     {
         var halfDuration = duration / 2;
 
         setTimeout(function()  
         {
             var element = document.getElementById(elementId);
-            element.style.opacity = min;
+            element.style.opacity = 0;
```
Comment (Syntasu) [inline js/synn.js]: Please so not change my code, its configurable for a reason.

File: js/synn.js [hunk 10]
```diff
@@ -5,21 +5,22 @@ var lib =
...
 
             setTimeout(function()  
             {
-                element.style.opacity = max;
+                element.style.opacity = 1;
```
Comment (Syntasu) [inline js/synn.js]: Same here.

File: js/synn.js [hunk 11]
```diff
@@ -5,21 +5,22 @@ var lib =
...
 
             }, halfDuration);  
 
         }, halfDuration);
     },
+
```
Comment (Syntasu) [inline js/synn.js]: Whitespace, please remove.

File: js/music-controls.js (hunk 8 above)
Comment (Syntasu) [reply js/music-controls.js]: Instead of assigning src directly, use element.setAttribute

Comment (Syntasu) [reply js/music-controls.js]: width 100% and height 100% should be in css, since it's constant.

File: index.html [hunk 12]
```diff
@@ -46,4 +47,7 @@
     <script src="js/music-handler.js"></script>
     <script src="js/music-controls.js"></script>
     <script src="js/background-handler.js"></script>
-</html>
\ No newline at end of file
+    <script src="js/jquery.js"></script>
+    <script src="js/jqueryui.js"></script>
+    <script src="js/cycle.js"></script>
+    </html>
```
Comment (Syntasu) [inline index.html]: Remove libraries.

File: index.html [hunk 13]
```diff
@@ -7,9 +7,10 @@
     </head>
 
     <body>
-        <img id="background"/>
+            <img id="cursor" src="img/cursor.png"/>
+        <div id ="slider"></div>
```
Comment (Syntasu) [inline index.html]: What is the purpose of #slider.
Name things properly please.

File: css/index.css [hunk 14]
```diff
@@ -22,8 +22,7 @@ body
     height: 100%;
 
     z-index: -1;
-    opacity: 0.3;
-    transition: opacity 0.3s; 
+
```
Comment (Syntasu) [inline css/index.css]: Whitespace.

File: index.html (hunk 13 above)
Comment (vecchiotom) [reply index.html]: the purpose of slider is just to hold all the backgrounds, it gets populated upon loading the page

Comment (Syntasu) [reply index.html]: Then please rename it to background-container.

File: js/music-controls.js (hunk 8 above)
Comment (vecchiotom) [reply js/music-controls.js]: that is not on the on mouse move, that's on the onwindowsload event

Comment (Syntasu) [reply js/music-controls.js]: Ok, please fix the indentation though.
